Modules/IO/MeshGifti/test/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+itk_module_test()
+
+set(ITKIOMeshGiftiTests
+  itkMeshFileReadWriteTest.cxx
+)
+
+CreateTestDriver(ITKIOMeshGifti "${ITKIOMeshGifti-Test_LIBRARIES}" "${ITKIOMeshGiftiTests}" )
+
+itk_add_test(NAME itkMeshFileReadWriteTestGifti1
+      COMMAND ITKIOMeshGiftiTestDriver itkMeshFileReadWriteTest
+      DATA{Baseline/white.gii}
+      ${ITK_TEST_OUTPUT_DIR}/white.gii
+)
+itk_add_test(NAME itkMeshFileReadWriteTestGifti2
+      COMMAND ITKIOMeshGiftiTestDriver itkMeshFileReadWriteTest
+      DATA{Baseline/aparc.gii}
+      ${ITK_TEST_OUTPUT_DIR}/aparc.gii
+)
+
+itk_python_expression_add_test(NAME itkGiftiMeshIOPythonTest
+  EXPRESSION "io = itk.GiftiMeshIO.New()")
```

Comment:
@fbudin69500 this is the most basic Python test.

Reply:
@dzenanz : Thanks for pointing out where the test is. For some reason I am pretty sure this was not visible in my browser earlier. Maybe I just needed to refresh my page. Looks good enough to test that the wrapping was successful.